spec.md

```diff
@@ -836,6 +842,8 @@ The following comparison assumes BBS signatures with curve BLS12-381, targeting
 
 For 128 bits security, ECDSA with curve P-256 takes 37 and 79 micro-seconds to sign and verify signature on a modern computer. BBS 680 and 1400 milliseconds to sign and verify a single message. However, ECDSA can only sign a single message whereas BBS can sign any number of messages at the expense of a bigger public key. To sign and verify 10 messages takes 3.7 and 5.4 milliseconds, and 22.3 and 24.4 milliseconds for 100 messages.
 
+(FIXME: How is sign and verify of 10 messages faster than a single message? Also, it seems to scale linearly, as would an ECDSA signature over 10/100 messages)
```

Comment:
I think those 680 and 1400 numbers are also in microseconds not milliseconds?

Reply:
It also kind of implies that there is space time tradeoff and that BBS can sign any number of messages in 680/1400ms. So maybe the 3.7 and 5.4 actually refer to ECDSA. Not sure.

Reply:
Sure feel free to remove it.

Reply:
@schanzen, could you update your PR to just remove this section entirely from the appendix?